.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Let's add newline chars to make GitHub happy as our dev is mostly here. Will just comment newline on the others.